### PR TITLE
fix(ui) Make tooltip on search results stats summary clearer

### PR DESCRIPTION
**Pull request by chriscollins3456**
We've heard that the tooltip for the "changed <time ago>" on search results cards is not helpful. This makes things a bit clearer.

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx [hunk 1]
```diff
@@ -74,12 +66,10 @@ export const DatasetStatsSummary = ({
...
             <Popover
                 content={
-                    <div>
-                        Changed on {toLocalDateTimeString(lastUpdatedMs)}.{' '}
-                        <Tooltip title="The time at which the data was last changed in the source platform">
-                            <HelpIcon />
-                        </Tooltip>
-                    </div>
+                    <PopoverContent>
+                        The time at which the data was last changed in the source platform:{' '}
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx:70]: Can we make this more concise? How about:

"Data was last changed in the source platform at 5/8/2021, 07:16PM EDT"

File: datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx [hunk 2]
```diff
@@ -1,14 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx:2]: Want to make this `/macro` while you're at it?

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM but would strongly suggest we make the wording even more concise

File: datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx (hunk 1 above)
Comment (chriscollins3456) [reply datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx:70]: i like this better too!

Comment (chriscollins3456) [reply datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx:70]: maybe change "at" -> "on" -- "Data was last changed in the source platform on 5/8/2021, 07:16PM EDT"

**Pull request merged**